promotion/aggregation.go

```diff
@@ -0,0 +1,64 @@
+package promotion
+
+import (
+	"time"
+
+	uuid "github.com/satori/go.uuid"
+	"github.com/shopspring/decimal"
+)
+
+var (
+	// ExcludedPromotions a map of promotions to be excluded from the aggregation calculation
+	ExcludedPromotions = map[string]bool{
```

Comment:
Do we really need this? Doesn't this correspond to having an `approximate_value` equal to the `bonus`?

Reply:
yeah, we could do that instead.

Reply:
did we want to just add in the `BonusPromotions` map into the column as well?